data/docs_markdown/Monthly_Token_Emission_Schedule.md

```
@@ -0,0 +1,130 @@
+  * Total Additional Emissions: 300,000,000 tokens over 20 years
+  * Final Cap: 800,000,000 tokens
+  * Disinflation Frequency: Monthly
+  * Starting Inflation Rate: 8% per year (0.667% per month)
+  * Disinflation Factor: ~0.989849199 per month
+  * Disinflation periods: 240 (20 years × 12 months)
+
+
+The monthly emission schedule follows a disinflationary curve, meaning that the inflation rate decreases over time. It starts at 8% annually and gradually reduces each month until it reaches near-zero inflation at the end of 20 years.
+### [](https://docs.io.net/docs/ionet-monthly-token-emission-schedule#monthly-emission-schedule)
+The table below demonstrates the first 12 months of emissions under the proposed schedule:
+#### Month
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)

(MD045, no-alt-text)